index.js

```diff
@@ -63,6 +63,12 @@ function render(state){
       ${Footer(state)}
     `;
 
+    document.querySelector('#navigation').addEventListener('click', (event) => {
```

Comment:
This prevents the default click event on the navigation _container_... how would you do this for the anchor tags _inside_ the navigation component?